### ISSUE TITLE
fileSuccess response is string?

### ISSUE THREAD

**Issue by vlatkodrljin** [flowjs/flow.js#83]
http://stackoverflow.com/questions/28724395/flow-js-filesuccess-response-is-string

**Comment by AidasK** [comment]
Yes, string is for compatibility reasons  and it makes library a bit smaller and a bit more flexible. However JSON is used everywhere and should be supported, so feel free to open a pull request for JSON parsing. I think it should be enabled through config.

**Comment by vlatkodrljin** [comment]
Feel free to add this in next improvement or in new version.
